pages/TheExplorationLibrary.html

```diff
@@ -36,7 +34,7 @@
     the gallery shows only mathematics explorations.
   </p>
 
-  <img src="static/images/galleryCategoryHighlighted.png" width="100%">
+  <img src="static/images/libraryCategoryHighlighted.png" width="100%">
```

Comment:
Shouldn't this picture show the user selecting "Mathematics" from the category dropdown, rather than their typing "Mathematics" into the search bar?